### PR TITLE
Embed video on homepage

### PR DESCRIPTION
**Pull request by musangowope**
- Add english and french video iframes to home. English video shows by default. Links English and French are below video to switch between english and french
DEMO:
https://www.loom.com/share/6cff48989dfa4350a571d5c2b212f3c0

resolves https://github.com/laws-africa/laws-africa.github.io/issues/43

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,9 +6,33 @@
 <header class="py-5 bg-light">
   <div class="container">
     <img src="/img/logo.png" class="img-fluid mb-5 d-block mx-auto" alt="logo" style="max-height: 150px;"/>
-    <h2 class="display-5 mb-5">
+    <h2 class="mb-display-5 mb-4">
```
Comment (longhotsummer) [inline index.html:9]: `mb-display`?

**Comment by longhotsummer** [comment]
Nice use of bootstrap tabs to toggle :)

File: index.html (hunk 1 above)
Comment (musangowope) [reply index.html]: removed

**Comment by musangowope** [comment]
> Pls take a look at https://getbootstrap.com/docs/5.0/helpers/ratio/ -- we need the iframe to be responsive, otherwise the mobile view it's truncated on the sides.

Note we are currently using bootstrap 4.4.1, meaning we don't have access to `ratio` utility classes so I created my own class to wrap around the iframes and enforce aspect ratio of `16:9`

https://www.loom.com/share/b56d642e752e43bab4fba8aa58277a7a

**Comment by longhotsummer** [comment]
> > Pls take a look at https://getbootstrap.com/docs/5.0/helpers/ratio/ -- we need the iframe to be responsive, otherwise the mobile view it's truncated on the sides.
> 
> Note we are currently using bootstrap 4.4.1, meaning we don't have access to `ratio` utility classes so I created my own class to wrap around the iframes and enforce aspect ratio of `16:9`
> 
> https://www.loom.com/share/b56d642e752e43bab4fba8aa58277a7a

V4 has https://getbootstrap.com/docs/4.0/utilities/embed/

**Comment by longhotsummer** [comment]
But your solution works great!

**Review by longhotsummer — APPROVED** [review]
Ship it! 🚢

**Pull request merged**